kubernetes/build.gradle

```diff
@@ -18,6 +18,24 @@ ext {
   outputFilePython = file("$buildDir/dist/Dockerfile-Python")
   outputFileR = file("$buildDir/dist/Dockerfile-R")
   outputFileScala = file("$buildDir/dist/Dockerfile-Scala")
+  outputFileExternalBackend = file("$buildDir/dist/Dockerfile-External-backend")
+}
+
+task createExternalBackendDockerfile(type: Dockerfile) {
+  // Based on h2o-3/docker/public/Dockerfile-h2o-release
+  // After official H2O image is out, remove duplication as mentioned in https://0xdata.atlassian.net/browse/SW-2294
+  destFile = outputFileExternalBackend
+  from "openjdk:11"
+  runCommand "mkdir -p /opt/h2oai/h2o-3/"
+  copyFile("h2o.jar", "/opt/h2oai/h2o-3/")
+  copyFile("sparkling-water-assembly-extensions_2.11-${version}-all.jar", "/opt/h2oai/h2o-3/")
+  defaultCommand(
+    "java",
+    "-cp",
+    "/opt/h2oai/h2o-3/sparkling-water-assembly-extensions_2.11-${version}-all.jar:/opt/h2oai/h2o-3/h2o.jar",
+    "-XX:+UseContainerSupport",
+    "-XX:MaxRAMPercentage=50",
```

Comment:
why this option is needed? XGBoost?

Reply:
For this, you need to ask @Pscheidl. I'm just copying the Dockerfile statements for now.

Once H2O official docker image is released, I will change this part of code to extend the h2o image

Reply:
This part is tricky. :cry: 
    "-XX:+UseContainerSupport" enables support for proper memory detection in containerized environment. This is a no brainer. However, the amount of memory is tricky. Normally, I'd give 90% to H2O. But if there is XGBoost, 50% is a good start, but it might need fine-tuning. We need to make this clear in our documentation as well. When users do not use XGBoost, huge amount of RAM is wasted. In the end, we agreed on 50%.

What I think is the proper way to do things is this: Once external XGBoost cluster is ready, H2O could spawn it on demand (or Sparkling water, or any external tool). Then, we might set this to a higher number by default.

Reply:
Thanks for the explanation @Pscheidl !